examples/SubscribedListenerLaravel.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
```

Comment:
I don't really understand the purpose of this file.
It looks like a front controller, but a lot of the dependencies are missing.
Also cross cutting concerns should generally not go into a front controller.